### PR TITLE
testing automation

### PR DESCRIPTION
**Pull request by kikobr82**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: scripts/dashboard/dashboard.sh [hunk 1]
```diff
@@ -41,15 +41,15 @@ do
 done
 cat ${BASEDIR}/students.txt|while read STUDENT
```
Comment (denstorti) [inline scripts/dashboard/dashboard.sh:42]: As you moved `students.txt` to dashboard/, can you fix `README.ME`, L276?

**Comment by denstorti** [comment]
Great idea!

File: scripts/dashboard/dashboard.sh (hunk 1 above)
Comment (kikobr82) [reply scripts/dashboard/dashboard.sh:42]: done.

**Comment by kikobr82** [comment]
Added some code to also update the progressiondata sheet, which auto update the progression chart.

**Comment by kikobr82** [comment]
> looks good
> 
> I'm sure you have tried to solve below, hence I approved the PR already:
> 
> can't we get the students name from github students group name using API?
> same for labs, can't get just reading folders names?

we should be able to get the list of students from the GH. haven't tried it
For the labs, GH is probably not the right place as we have exercises that are no longer required.

Maybe, for the time, the worksheet might be the best place to collect the lab information instead of the files in the repository ad we can remove it from the repo.

**Pull request merged**